Makefile.kokkos

```diff
@@ -1312,6 +1312,8 @@ ifneq ($(KOKKOS_INTERNAL_NEW_CONFIG), 0)
 endif
 
 KOKKOS_HEADERS += $(wildcard $(KOKKOS_PATH)/core/src/*.hpp)
+KOKKOS_HEADERS += $(wildcard $(KOKKOS_PATH)/core/src/View/*.hpp)
+KOKKOS_HEADERS += $(wildcard $(KOKKOS_PATH)/core/src/View/MDSpan/*.hpp)
```

Comment:
Why do we need these?  It looks like we are doing `#include <View/Kokkos_FooBar.hpp>`

Reply:
file dependencies for rebuilds.